lua/terrortown/lang/en/infected.lua

```diff
@@ -5,16 +5,16 @@ L[INFECTED.name] = "Infected"
 L[INFECTED.defaultTeam] = "Team Infected"
 L["hilite_win_" .. INFECTED.defaultTeam] = "TEAM INFECTED WON"
 L["win_" .. INFECTED.defaultTeam] = "The Infected has won!"
-L["info_popup_" .. INFECTED.name] = [[Now its your turn! Infect them all by killing them.]]
+L["info_popup_" .. INFECTED.name] = [[Now it's your turn! Infect them all by killing them.]]
 L["body_found_" .. INFECTED.abbr] = "They were an Infected!"
 L["search_role_" .. INFECTED.abbr] = "This person was an Infected!"
 L["ev_win_" .. INFECTED.defaultTeam] = "The ill Infected won the round!"
 L["target_" .. INFECTED.name] = "Infected"
-L["ttt2_desc_" .. INFECTED.name] = [[The Infected needs to infect every player to win. He will infect other players by killing them.
-If a player gets infected, the player looks like a zombie and is also able to infect other players. So you can build up a whole army.
-But there is one thing you need to get in mind: If the host (the main infected player with a normal model) will die or disconnect, each player that the host infected will die.
+L["ttt2_desc_" .. INFECTED.name] = [[The Infected needs to infect every player to win. They infect other players by killing them.
+If a player gets infected, the player becomes a zombie and is able to infect other players. This allows zombies to build up a whole army.
+However, there is one thing you need to get in mind: If the host (the main infected player with a normal model) dies or disconnects, each player that the host directly infected will die.
```

Comment:
```suggestion
However, there is one thing you need to keep in mind: If the host (the original, non-zombie infected player) dies or disconnects, each player that the host directly infected will die.
```
Also, don't ALL infected players die in that scenario? Do zombies who where infected by another zombie really not die?

Reply:
yes all zombies die. I'm 99% sure that "second grade" zombies die as well

Reply:
thx, updated suggestions accordingly

Reply:
Excellent, I agree with these changes.